src/openboardview/FileFormats/GenCADFile.cpp

```diff
@@ -248,13 +251,14 @@ bool GenCADFile::parse_shape_pins_to_component(
 				if (pos_ast && pin_name_ast) {
 					BRDPin pin;
 					pin.radius = 0.5;
-					// enable the code below once the pin.radius will be processed
-					/*mpc_ast_t *padstack_name_ast = mpc_ast_get_child(pin_ast, "pad_name|nonquoted_string|regex");
+					mpc_ast_t *padstack_name_ast = mpc_ast_get_child(pin_ast, "pad_name|nonquoted_string|regex");
+					mpc_ast_t *padstack_ast = 0;
 					if (padstack_name_ast) {
-					    mpc_ast_t *padstack_ast = get_padstack_by_name(padstack_name_ast->contents);
-					    if (padstack_ast)
-					        pin.radius = get_padstack_radius(padstack_ast);
-					}*/
+					    padstack_ast = get_padstack_by_name(padstack_name_ast->contents);
+					// enable the code below once the pin.radius will be processed
+					//    if (padstack_ast)
+					//       pin.radius = get_padstack_radius(padstack_ast);
```

Comment:
This comment block starting `//` seems to have a bit misleading indent level (logically inside if, visually outside if).

And a minor thing: this source file formatting was done via clang-format-15 default parameters as suggested in 5. https://github.com/OpenBoardView/OpenBoardView/pull/219#issuecomment-1146615659

That formatter use 1-tab indent, but the introduced code hax tab-and-spaces mix - here 
![image](https://user-images.githubusercontent.com/1609972/176180111-0b544de5-0d5d-4c47-962a-76ba4a6a2bb7.png)

Reply:
Thanks for reviewing and catching that